thorlcr/activities/hashdistrib/thhashdistribslave.cpp

```diff
@@ -3168,6 +3195,16 @@ class HashDedupSlaveActivityBase : public CSlaveActivity
             }
             else
             {
+                // end of input from phase
+
+                // For testing only: spill one bucket. NB: this is in effect before finished reading input. i.e. the buckets are not yet marked as completed
+                if (testSpillTimes)
+                {
+                    bucketHandler->spillBucket(false);
+                    testSpillTimes--;
+                }
+
+                bucketHandler->checkCompletedBuckets();
```

Comment:
NB: test is now spilling before and after buckets being marked complete, to ensure it is testing reading back spilt bucket streams.